docs/architecture-decision-records/002-component-constuctors.md

```diff
@@ -46,21 +46,21 @@ Constructors should follow the following rules for consistency.
 
    ```ts
    class MyConstruct {
-     constructor(scope: Stack, id: string) {
+     constructor(scope: Stack) {
        ...
      }
    }
    ```
 
-2. They can also take a `props` object which should be correctly typed
+   The construct/pattern will then have a static `id` as it will never change, for example the `Stage` parameter.
+
+2. They should all accept a `scope` of type `GuStack` and `props` of a custom type:
```

Comment:
I don't think that "should _all_" is quite true? I think the original phrasing ("They can also take a `props` object which should be correctly typed"), is probably still fine here?

Reply:
👍🏽  have reverted that bit and force pushed.